packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -510,9 +535,10 @@ export class ProjectService {
                 'Warehouse credentials must be provided to connect to your dbt project',
             );
         }
-        const warehouseClient = warehouseClientFromCredentials(
-            project.warehouseConnection,
-        );
+        const warehouseClient =
+            this.projectModel.getWarehouseClientFromCredentials(
```

Comment:
should we use `_getWarehouseClient` here?

Reply:
Yep! Ty